video/out/opengl/context_glx.c

```diff
@@ -88,7 +88,7 @@ static bool create_context_x11(struct ra_ctx *ctx, GL *gl, bool es)
         return false;
     }
 
-    if (!strstr(glxstr, "GLX_ARB_create_context") ||
+    if (!gl_check_extension(glxstr, "GLX_ARB_create_context") ||
```

Comment:
Here and in the WGL case there is the extension and "_profile" variant of it. Didn't look at the code closely enough to judge which is the correct - just the main or both extension should be checked.

Do you mind having a look and adding a note in the commit message and/or inline?

Reply:
Good catch.
It appears mpv should check for `GLX_ARB_create_context_profile` instead since it uses the profile mask attribute.

    If GLX_ARB_create_context_profile is not supported, then the
    GLX_CONTEXT_PROFILE_MASK_ARB attribute, the
    GLX_CONTEXT_CORE_PROFILE_BIT_ARB and
    GLX_CONTEXT_COMPATIBILITY_PROFILE_BIT_ARB bits in that attribute,
    and the ERROR_INVALID_PROFILE_ARB error are not defined, and
    specifying the attribute in <attribList> attribute will generate
    BadValue.
https://www.khronos.org/registry/OpenGL/extensions/ARB/GLX_ARB_create_context.txt